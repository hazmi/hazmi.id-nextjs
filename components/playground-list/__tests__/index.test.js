import React from 'react'
import renderer from 'react-test-renderer'
import PlaygroundList from '../index'

describe('With Snapshot Testing', () => {
  it('show the correct content', () => {
    const component = renderer.create(<PlaygroundList />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})