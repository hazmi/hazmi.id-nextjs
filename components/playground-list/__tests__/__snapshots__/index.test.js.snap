// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing show the correct content 1`] = `
<div
  className="playground-list"
>
  <section
    className="playground-list__item"
  >
    <h3
      className="playground-list__item-header"
    >
      <a
        href="https://todomvc.hazmi.now.sh/"
      >
        TodoMVC - Next.js
      </a>
    </h3>
    <div
      className="playground-list__item-content"
    >
      <div
        className="playground-list__item-image"
      >
        <img
          src="/static/playground/todo-nextjs.jpg"
          width="100%"
        />
      </div>
      <p>
        Another TodoMVC app. Written using 
        <a
          href="https://nextjs.org/"
        >
          Next.js
        </a>
         framework.
      </p>
    </div>
    <div
      className="playground-list__item-action"
    >
      <a
        className="playground-list__item-readme"
        href="https://github.com/hazmi/todomvc-nextjs/blob/master/README.md"
      >
        Readme
      </a>
      <a
        className="playground-list__item-source"
        href="https://github.com/hazmi/todomvc-nextjs"
      >
        Source
      </a>
      <a
        className="playground-list__item-demo"
        href="https://todomvc.hazmi.now.sh/"
      >
        Demo
      </a>
    </div>
  </section>
  <section
    className="playground-list__item"
  >
    <h3
      className="playground-list__item-header"
    >
      <a
        href="https://cra-todomvc.netlify.com/#/"
      >
        TodoMVC - Create React App
      </a>
    </h3>
    <div
      className="playground-list__item-content"
    >
      <div
        className="playground-list__item-image"
      >
        <img
          src="/static/playground/todo-cra.jpg"
          width="100%"
        />
      </div>
      <p>
        Another TodoMVC app. Written using 
        <a
          href="https://facebook.github.io/create-react-app/"
        >
          Create React App
        </a>
         framework.
      </p>
    </div>
    <div
      className="playground-list__item-action"
    >
      <a
        className="playground-list__item-readme"
        href="https://github.com/hazmi/todomvc-cra/blob/master/README.md"
      >
        Readme
      </a>
      <a
        className="playground-list__item-source"
        href="https://github.com/hazmi/todomvc-cra"
      >
        Source
      </a>
      <a
        className="playground-list__item-demo"
        href="https://cra-todomvc.netlify.com/#/"
      >
        Demo
      </a>
    </div>
  </section>
</div>
`;
